### PR TITLE
client: add API for GET /systems/<label>

### PR DESCRIPTION
**Pull request by mvo5**
This commit implements the `client.SystemDetails()` call that will
return the data from the `/systems/<label>/` get endpoint.

### CODE REVIEW

File: client/systems.go [hunk 1]
```diff
@@ -138,3 +139,27 @@ func (client *Client) RebootToSystem(systemLabel, mode string) error {
 	}
 	return nil
 }
+
+type SystemDetailsData struct {
```
Comment (pedronis) [inline client/systems.go:143]: this name probably doesn't need Data at the end

**Review by pedronis — APPROVED** [review]
looks good, one naming comment

**Review by Meulengracht — APPROVED** [review]
LGTM

File: client/systems.go (hunk 1 above)
Comment (mvo5) [reply client/systems.go]: I removed this

**Pull request merged**